crates/bitwarden-exporters/src/lib.rs

```diff
@@ -132,3 +134,12 @@ pub struct Identity {
     pub passport_number: Option<String>,
     pub license_number: Option<String>,
 }
+
+pub struct SshKey {
+    /// Ssh private key (ed25519/rsa) in unencrypted openssh private key format https://github.com/openssh/openssh-portable/blob/master/PROTOCOL.key
+    pub private_key: Option<String>,
+    /// Ssh public key (ed25519/rsa) according to RFC4253 https://datatracker.ietf.org/doc/html/rfc4253#section-6.6
```

Comment:
There are multiple ways to encode public keys, are we using the OpenSSH format or [RFC 4716](https://datatracker.ietf.org/doc/html/rfc4716#section-3)?  4716 includes headers and are multi line, while OpenSSH is just the B64 in a single line.

Reply:
We use the single line openssh format. This is what 1Password does too, and what servers and GitHub accept as public keys pasted as authentication keys.